src/index.js

```diff
@@ -10,7 +10,7 @@ const converters = require('./converters.js')
 
 var input, output
 
-const version = JSON.parse(fs.readFileSync('./package.json')).version
+const version = require('../package.json').version
```

Comment:
Why the now different path?

Reply:
The path looks for one folder above since (on my machine), the relaxed commands points to `/usr/local/lib/node_modules/relaxedjs/src/index.js` while `package.json` is located in `/usr/local/lib/node_modules/relaxedjs/package.json`.

I tested it locally to make sure if it works. Let me know if there are problems with this, and I will try to test it in more situations.

Reply:
You are right, the binary points to src/index.js
https://github.com/RelaxedJS/ReLaXed/blob/master/package.json#L7